polyfills/URL/detect.js

```diff
@@ -0,0 +1,25 @@
+(function (global) {
+	/*
+	 * Browsers may have:
+	 * No global URL object
+	 * URL with static methods only - may have a dummy constructor
+	 * URL with members except searchParams
+	 * Full URL API support
+	 */
+	'use strict';
+
+	try {
+		if (global.URL) {
+			var nativeURL = new global.URL('http://example.com');
+
+			if ('searchParams' in nativeURL) {
+				return true;
+			}
+
+			if (!('href' in nativeURL)) {
+				return false;
+			}
+		}
+	}
+	catch (error) { }
+}(this))
```

Comment:
```js
(function (global) {
	/*
	 * Browsers may have:
	 * No global URL object
	 * URL with static methods only - may have a dummy constructor
	 * URL with members except searchParams
	 * Full URL API support
	 */
	'use strict';

	try {
		if (global.URL) {
			var nativeURL = new global.URL('http://example.com');

			if ('searchParams' in nativeURL) {
				return true;
			}

			if (!('href' in nativeURL)) {
				return false;
			}
		}
	}
	catch (error) { }

	return true;
}(this))
```

Works fine for me, thanks.

But, where is my other tests?

![1422968312421](https://cloud.githubusercontent.com/assets/803674/6020507/c6b6df80-abbd-11e4-91cd-62a4d8e9d163.png)